hypothesis-python/src/hypothesis/searchstrategy/strategies.py

```diff
@@ -268,6 +274,14 @@ def example(self, random=not_set):
         if random is not not_set:
             note_deprecation("The random argument does nothing", since="2019-07-08")
 
+        if getattr(sys, "ps1", None) is None:  # pragma: no branch
+            warnings.warn(
+                "The `.example()` method is for interactive use only.  It's good for "
+                "exploring strategies, but `@given` is much better for tests - better "
+                "performance, minimal failures, deduplication, etc. (%r)" % (self,),
```

Comment:
I’m not in love with the "etc" here. It feels like we're expecting the user to know about the Hypothesis internals.

Do we have a docs page or anything we can link people to that explains what the difference between `@given` and `.example()` is?